src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -1556,7 +1567,34 @@ public boolean hasAlly(Town town) {
 	public boolean hasMutualAlly(Town town) {
 		return hasAlly(town) && town.hasAlly(this);
 	}
+	/**
+	 * Only to be used when loading the database.
+	 * @param towns List&lt;Town&gt; which will be loaded in as trusted towns.
+	 */
+	public void loadTrusted(List<Town> towns) {
```

Comment:
This method is not really needed imo, can just use addTrustedTown in a loop during loading

Reply:
This is similar to how we load Allies, it can be good to have separate methods only for loading. It also makes for cleaner looking loading code.

Reply:
I don't feel too confident throwing my weight behind either or but I will say I implemented it similar to how allies are loaded.